apps/web-giddh/src/assets/css/style-1.scss

```diff
@@ -401,7 +401,9 @@ aside-menu-account .container-fluid {
 .pd {
     padding: 5px;
 }
-
+.pd-0{
```

Comment:
no need to add pd-0 for padding:0 , class **p-0** is already used globally for padding:0 @shalinee-walkover

Reply:
removed